README.md

```diff
@@ -77,6 +77,26 @@ You can find more PayPal configuration options in the [Braintree JS client SDK v
 
 The structure of the PayPal payment method object returned in the callback of `requestPaymentMethod` can be found [here](http://braintree.github.io/braintree-web/current/PayPal.html#~tokenizePayload).
 
+### PayPal Credit
+
+PayPal Credit can also be enabled in Drop-in by including `paypalCredit` configuration options in the `create` call:
+
+```js
+braintree.dropin.create({
+  authorization: 'CLIENT_AUTHORIZATION',
+  selector: '#dropin-container',
+  paypalCredit: {
+    flow: 'checkout', // Required for PayPal Credit
+    amount: 10.00,
+    currency: 'USD'
+  }
+}, callback);
+```
+
+PayPal Credit configuration parameters are the same as those for [PayPal](https://braintree.github.io/braintree-web/current/PayPal.html#tokenize).
```

Comment:
Should we link to the PayPal Checkout component. There is a very slight difference here.